pkg/cache/clusterqueue.go

```diff
@@ -113,6 +123,27 @@ type ResourceQuota struct {
 	LendingLimit   *int64
 }
 
+func (r *ResourceQuota) nominalSafe() int64 {
```

Comment:
Since we do a root walk to determine capacity, a node in path to root may not define quota for the `FlavorResource` we're checking. These convenience functions allow us to omit the nil check